project/PekkoDependWalker.scala

```diff
@@ -30,4 +32,10 @@ object PekkoDependWalker {
         ScopeKeyMatcher((Compile / packageBin).scopedKey, CheckBoth),
         ScopeKeyMatcher((CompileJdk9 / compile).scopedKey, CheckConfig))))
 
+  lazy val protobufV3Settings = Seq(
+    walkTasks := Seq(
+      WalkTask(
+        ScopeKeyMatcher(OsgiKeys.bundle.scopedKey, CheckTask),
+        ScopeKeyMatcher(AssemblyKeys.assembly.scopedKey, CheckTask))))
```

Comment:
For modules that have applied this setting, it will enforce this:

> When `OsgiKeys.bundle` executes, the `AssemblyKeys.assembly` will be before  `OsgiKeys.bundle` and execute too.